components/developer/gdb/Makefile

```diff
@@ -47,9 +47,6 @@ COMPONENT_PROJECT_URL=	https://www.gnu.org/software/gdb/
 
 CONFIGURE_DEFAULT_DIRS=no
 
-# Set python version used by this product
-PYTHON_VERSION=3.7
-
 include $(WS_MAKE_RULES)/common.mk
 
```

Comment:
I recommend to add
PYTHON_VERSIONS= $(PYTHON_VERSION)
for packages that only need or want a single python version. Adding this will eg. change pkg5.

Reply:
Yes, adding that should change the `pkg5` file, but otherwise everything else should be same.  OTOH, for `setup.py` build styles such line would cause shorter build times because only single version will be built instead of all of them.

Maybe we should try to make this default for non-setup.py builds?  And similarly with `PERL_VERSIONS` too?

Reply:
Yes, that would be a nice enhancement.

Reply:
Here it is: PR #9653